test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/KeyValuePairModelBinderTest.cs

```diff
@@ -212,21 +208,13 @@ private static DefaultModelBindingContext CreateContext()
             IValueProvider valueProvider,
             Type keyValuePairType)
         {
-            var metataProvider = new EmptyModelMetadataProvider();
+            var metadataProvider = new TestModelMetadataProvider();
```

Comment:
Does changing from an `EmptyModelMetadataProvider` to a `TestModelMetadataProvider` mean anything?